pkg/minikube/cluster/cluster.go

```diff
@@ -224,6 +225,12 @@ func createDriverOptions(driver drivers.Driver, explicitOptions map[string]inter
 
 func (m *MachineConfig) CacheMinikubeISOFromURL() error {
 	fmt.Println(fmt.Sprintf("Downloading ISO '%s'", m.MinikubeISO))
+	tmpISOFile, err := ioutil.TempFile("", "tmpISOFile")
+	fmt.Println("TMP FILE IS", tmpISOFile.Name())
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(tmpISOFile.Name())
 
 	// store the iso inside the MINISHIFT_HOME dir
 	response, err := http.Get(m.MinikubeISO)
```

Comment:
I think we do have it - https://github.com/minishift/minishift-b2d-iso/releases, https://github.com/minishift/minishift-centos-iso/releases